cpp/dolfinx/io/xdmf_utils.cpp

```diff
@@ -57,6 +57,7 @@ xdmf_utils::get_cell_type(const pugi::xml_node& topology_node)
          {"quadrilateral_9", {"quadrilateral", 2}},
          {"quadrilateral_16", {"quadrilateral", 3}},
          {"hexahedron", {"hexahedron", 1}},
+         {"wedge", {"prism", 1}},
```

Comment:
When did 'prism' enter DOLFINx? No a good name since more than one 2D cell shape is a prism, see https://en.wikipedia.org/wiki/Prism_(geometry).